config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+class Rack::Attack
+  throttle('feedback/ip', limit: 5, period: 1.hour) do |req|
+    req.ip if req.path == '/v0/feedback' && req.post?
```

Comment:
Just want to make sure `req.ip` appears as the end-user's IP address, not e.g. the TIC address?

Reply:
👍, tested in dev to confirm that the IP is the client address.

Reply:
Does monitoring & alerting exist if we start returning 90% `429` responses?  I'm worried that some deployment/proxy  setting could change in the future such that `req.ip` stops representing client IP address and clients get effectively blocked from using the API.

Reply:
@omgitsbillryan all of the error rate alerting is for 5xx responses.